### PR TITLE
Add Roslyn version 3.6.0-3.20218.3 binaries from the nuget `Microsoft.Net.Compilers.Toolset`

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: Microsoft.Net.Compilers/3.6.0/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-3.6.0-4.20224.5
\ No newline at end of file
+3.6.0-3.20218.3
```
Comment (akoeplinger) [inline Microsoft.Net.Compilers/3.6.0/version.txt:1]: just confirming it's intentional we're using a lower version right?

Comment (radical) [reply Microsoft.Net.Compilers/3.6.0/version.txt:1]: That's what xplat-master is using, which is from `.NET Core SDK 3.1.4xx`. The earlier version is used by `2019-12`, which got it from `.NET Core SDK 3.1.3xx`. I just rechecked, and these are still the versions being used.
Just that look OK?

Comment (akoeplinger) [reply Microsoft.Net.Compilers/3.6.0/version.txt:1]: TBH I stopped remembering how all these versions worked. If we shipped 2019-12 with that version we should probably use the same in xplat-master too.

Comment (radical) [reply Microsoft.Net.Compilers/3.6.0/version.txt:1]: Um we now get all the sdk+roslyn versions from maestro channel, so everything stays in sync. It doesn't sound right to tweak this one version. Do you want to maybe wait for some time to see if they update the versions on the channel?

2019-12 will continue to use the version it needs to, via the older commit reference.

Comment (akoeplinger) [reply Microsoft.Net.Compilers/3.6.0/version.txt:1]: If we get it from maestro then it's probably fine :)

**Pull request merged**